### PR TITLE
Updated badges and links

### PR DESCRIPTION
**Pull request by AshishMhrzn10**
I added badges, fixed a typo and also added the link for License which wasn't working in README.md.
Contributes to issue #202

### CODE REVIEW

**Comment by Tiagoperes** [comment]
Ty for the contribution! Please, don't mind the validation error, it's a problem on our end for external pull requests.

**Review by Tiagoperes — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # [Beagle Angular](https://docs.usebeagle.io/v/v1.0-en/get-started/installing-beagle/web#angular)
+
+<a href="CONTRIBUTING.md" alt="Hacktoberfest"><img src="https://badgen.net/badge/hacktoberfest/friendly/pink" /></a>
+ [![Open Source](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://opensource.org/) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/ZupIT/beagle-web-react/blob/master/LICENSE)
```
Comment (carolinegoncalveszup) [inline README.md:4]: Can you please fix the url to refer to beagle-web-angular instead of beagle-web-react?

File: README.md [hunk 2]
```diff
@@ -225,5 +229,4 @@ You should see the screen with the text present in the text attribute in the jso
 `Well done, you created your first screen with Beagle!`
 
 # License
-Beagle Angular is Apache licensed.
-<!--[Apache licensed](https://github.com/ZupIT/beagle-web-react/blob/master/LICENSE)-->
+Beagle Angular is developed under [Apache license 2.0](https://github.com/ZupIT/beagle-web-react/blob/master/LICENSE).
```
Comment (carolinegoncalveszup) [inline README.md:232]: The license url should also be fixed to refer to beagle-web-angular

**Comment by AshishMhrzn10** [comment]
@carolinegoncalveszup  Please check it. I have updated the issue.

**Review by carolinegoncalveszup — APPROVED** [review]

**Review by Tiagoperes — APPROVED** [review]

**Pull request merged**